### ISSUE TITLE
Connect and exchange messages on kusama

### ISSUE THREAD

**Issue by ryanchristo** [ChainSafe/gossamer#467]
## Expected Behavior

We should be able to connect to Kusama, exchange status messages, send a block request message and receive a block response message.

## Current Behavior

We previously attempted to connect to Alexander. See #227 for more details.

## Related Issues

- [ ] #443
- [ ] #521
- [x] #552 / #538
- [ ] #554
- [ ] #563

**Comment by ryanchristo** [comment]
This is currently in progress / blocked by #554, #552/#538, #521, and #443...

**Comment by ryanchristo** [comment]
closing in favor of #580
